docs/ldml/tr35-keyboards.md

```diff
@@ -1022,12 +1021,32 @@ See  [`<displayOptions baseCharacter=…/>`](#element-displayoptions).
 
 ### Element: display
 
-The `display` element describes how a character, that has come from a `keys/key` element, should be displayed on a keyboard layout where such display is possible.
+The display elements can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element will be used for keytop display. However some characters, such as diacritics, do not display well on their own.
+
+Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.
+
+Finally, this provides a way to specify the keytop for keys which do not otherwise produce output, which is common for the `@layerId` attribute.
+
+Note: `displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` where needed.
+
+For combining characters, U+25CC `◌` is used as a base. It is an error to use a combining character without a base in the `display` attribute.
+
+For example, a key which outputs a combining tilde (U+0303) can be represented as follows:
+
+```xml
+    <display output="\u{0303}" display="◌̃" />  <!-- \u{25CC} \u{0303}-->
```

Comment:
```suggestion
    <display output="\u{0303}" display="◌̃" />  <!-- \u{25CC} \u{0303}-->
    <display output="\u{0303}" display="\u{25cc}\u{0303}" />  <!-- also acceptable -->
```

Given how badly it displays in GitHub preview monospace here, the current example may be counter-productive?

![image](https://github.com/unicode-org/cldr/assets/4498365/f8eac13a-73ed-44e2-b6ee-090fa5da361d)

Reply:
maybe, though it's just an example.